SConstruct

```diff
@@ -1735,6 +1744,8 @@ if doConfigure :
 
 		if "-DIECORE_WITH_FREETYPE" in imageEnv["CPPFLAGS"] :
 			imageEnv.Append( LIBS = "freetype" )
+			if env["PLATFORM"] == "win32":
+				imageEnv.Append( LIBS = ["libpng"] )
```

Comment:
It looks like FreeType uses PNG to "Support PNG compressed OpenType embedded bitmaps." I'm not too familiar with the fonts Gaffer and Cortex use but that sounds like something we could leave out. Do you agree?

Digging in the cmake for FreeType it turns out that if it can find the PNG library it will use it even if it's set to NO unless a CMAKE_DISABLE_FIND_PACKAGE is set for the PNG library.

I'll add that in the dependencies and that should remove the need for linking to PNG in Cortex.

Reply:
Yes, I'd be very surprised if we needed PNG compressed OpenType embedded bitmaps. If you don't mind, let's give it a go without and see what happens.

Reply:
Cool, I sorted out the PNG linking in the dependencies so this commit is dropped now.